setup.py

```diff
@@ -95,8 +100,8 @@ def checkversion(GEOS_dir):
 # Specify all the required mpl data
 # create pyproj binary datum shift grid files.
 pathout =\
-os.path.join('lib',os.path.join('mpl_toolkits',os.path.join('basemap','data')))
-if sys.argv[1] not in ['sdist','clean']:
+    os.path.join('lib', os.path.join('mpl_toolkits', os.path.join('basemap', 'data')))
```

Comment:
If you could, please restructure this assignment so that it doesn't use the backslash?